### DISCUSSION TITLE
请问如何下载视频（CQ:video）

### DISCUSSION THREAD

**Discussion by yyx2684264** [Mrs4s/go-cqhttp#2135] [Q&A] [answered]
各位大佬，目前在使用过程中遇到个问题，还望大家帮忙解答下：

在接受到视频消息后视频的CQcode，内容如下，但是通过CQcode中的url值访问视频会提示未发送任何数据导致下载失败，请问下我想下载视频到本地需要怎么做：
` [CQ:video,file=f43ed160f4f135741283825fc19ead30.video,url=http://49.86.42.118:80/qqdownload?ver=537154261&amp;rkey=3081fd0201010481f53081f2020101020100020440209e360481a63330353130323031303030343336333033343032303130303032303432646336316361623032303337613164626230323034626438336538373830323034363435383532626130343130663433656431363066346631333537343132383338323566633139656164333030323033376131616666303230313030303431343030303030303038363636393663363537343739373036353030303030303034333133303330333302046459ac8304350000000866696c657479706500000004313030330000000b646f776e656e63727970740000000130000000047175696300000001300400&amp;filetype=1003&amp;videotype=1&amp;subvideotype=0&amp;term=unknow]`

**Comment by Misaka-Mikoto-Tech** [comment]
同问，用各种工具都下载不了

**Comment by Misaka-Mikoto-Tech** [comment]
> 各位大佬，目前在使用过程中遇到个问题，还望大家帮忙解答下：
> 
> 在接受到视频消息后视频的CQcode，内容如下，但是通过CQcode中的url值访问视频会提示未发送任何数据导致下载失败，请问下我想下载视频到本地需要怎么做： ` [CQ:video,file=f43ed160f4f135741283825fc19ead30.video,url=http://49.86.42.118:80/qqdownload?ver=537154261&amp;rkey=3081fd0201010481f53081f2020101020100020440209e360481a63330353130323031303030343336333033343032303130303032303432646336316361623032303337613164626230323034626438336538373830323034363435383532626130343130663433656431363066346631333537343132383338323566633139656164333030323033376131616666303230313030303431343030303030303038363636393663363537343739373036353030303030303034333133303330333302046459ac8304350000000866696c657479706500000004313030330000000b646f776e656e63727970740000000130000000047175696300000001300400&amp;filetype=1003&amp;videotype=1&amp;subvideotype=0&amp;term=unknow]`

请问你现在找到下载方法了吗？

**Comment by Misaka-Mikoto-Tech** [comment]
顶一下

**Comment by Natrium0521** [comment, marked as answer]
把`url`里所有`&amp;`替换成`&`

**Reply by Misaka-Mikoto-Tech** [reply]
并不行

**Reply by Natrium0521** [reply]
行的 你可以重新获取一次CQ码 之前的url应该过期了
